README.md

```diff
@@ -22,7 +24,14 @@ console.log(results);
 ```
 
 ```js
-// Your answer...
+// var authorsController = {
+show(req){
+  Author.findOne({name: req.name}, (err, author) => {
+    console.log(author);
+  })
+}
+}
+authorsController.show({name: "bob"});
```

Comment:
The content of the route looks good, but you need to use `app.get(req, res){ ... }`, not `show(req)`